confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/input/ConfluenceXMLPackage.java

```diff
@@ -942,6 +980,42 @@ private void readPageObject(XMLStreamReader xmlReader)
         }
     }
 
+    private void readBlogPostObject(XMLStreamReader xmlReader)
+        throws XMLStreamException, ConfigurationException, FilterException
+    {
+        ConfluenceProperties properties = new ConfluenceProperties();
+
+        properties.setProperty(KEY_PAGE_BLOGPOST, true);
+
+        long pageId = readObjectProperties(xmlReader, properties);
+
+        savePageProperties(properties, pageId);
```

Comment:
I'm not sure this is a good idea, sounds much safer to index the blogs in their own location.

Reply:
Actually that one should not cause technical problems as long as the ids are not colliding, it's just very missleading when looking at the index files.